### DISCUSSION TITLE
Are there plans for Webstorm support?

### DISCUSSION THREAD

**Discussion by pauleustice** [github/vscode-github-actions#53] [General]
Hi! I'd like to know whether GitHub have any plans for a Webstorm (or indeed any Jetbrains IDEs) equivalent to this plugin?

In our org we do a lot of work with GH Actions but mostly work with Webstorm. Even just the ability to get schema / syntax highlighting would be a brilliant addition to our engineering process.

Cheers 👍

**Comment by Vadko** [comment]
➕

**Comment by KetchupOnMyKetchup** [comment]
We open sourced our [language services repo](https://github.com/actions/languageservices) and plan to get it into a (General Availability, 1.0) GA and stable state. Since we aren't at GA phase yet, there are still possible breaking changes that could happen, it would be better to wait until we hit that state before developing on top of our language services. We have hope that other developers will start to adopt our open source repo and can build can build extensions for other IDEs in the future. Thank you!

**Reply by dolunayglobal** [reply]
Reply
